hera_cal/io.py

```diff
@@ -626,6 +635,92 @@ def load_flags(flagfile, filetype='h5', return_meta=False):
         return flags
 
 
+def get_file_lst_range(filepaths, filetype='uvh5', add_int_buffer=False):
+    """
+    Get a file's start, stop and dlst in radians.
+
+    Start and stop are bin center.
+
+    Miriad standard is bin start, so shift by int_time / 2 is done.
+    UVH5 standard is bin center, so lsts are left untouched.
+
+    Parameters:
```

Comment:
This docstring has a different style from the rest. I don't really care, but maybe you do.